__tests__/app.test.js

```diff
@@ -169,3 +169,93 @@ describe("GET /api/articles", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        const commentsArray = body.comments;
+        expect(
+          commentsArray.every((comment) => comment.hasOwnProperty("author"))
+        ).toBe(true);
+        expect(
+          commentsArray.every((comment) => comment.hasOwnProperty("comment_id"))
+        ).toBe(true);
+        expect(
+          commentsArray.every((comment) => comment.hasOwnProperty("body"))
+        ).toBe(true);
+        expect(
+          commentsArray.every((comment) => comment.hasOwnProperty("created_at"))
+        ).toBe(true);
+        expect(
+          commentsArray.every((comment) => comment.hasOwnProperty("votes"))
+        ).toBe(true);
+        expect(
+          commentsArray.every((comment) => comment.hasOwnProperty("article_id"))
+        ).toBe(true);
+      });
+  });
+  test("responds with an comments array of objects sorted by date created", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        console.log(body.comments);
+        expect(body.comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("responds with the correct array of comment objects for a given article", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .then(({ body }) => {
+        expect(body.comments).toEqual([
+          {
+            comment_id: 15,
+            body: "I am 100% sure that we're not completely sure.",
+            votes: 1,
+            author: "butter_bridge",
+            article_id: 5,
+            created_at: "2020-11-24T00:08:00.000Z",
+          },
+          {
+            comment_id: 14,
+            body: "What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.",
+            votes: 16,
+            author: "icellusedkars",
+            article_id: 5,
+            created_at: "2020-06-09T05:00:00.000Z",
+          },
+        ]);
+      });
+  });
+  test("responds with a 404 code when passed an invalid article id", () => {
+    return request(app).get("/api/articles/50/comments").expect(404);
+  });
+  test("responds with appropriate error message when passed invalid article id", () => {
+    return request(app)
+      .get("/api/articles/50/comments")
+      .then(({ text }) => {
+        expect(text).toBe("No article found for article_id 50");
+      });
+  });
+  test("responds with a 404 code when passed an article id for an article with no comments", () => {
+    return request(app).get("/api/articles/8/comments").expect(404);
+  });
+  test("responds with appropriate error message when passed a valid article id with no comments", () => {
+    return request(app)
+      .get("/api/articles/8/comments")
+      .then(({ text }) => {
+        expect(text).toBe("No comments found for article_id 8");
+      });
+  });
+});
```

Comment:
Make sure to do a test for an invalid comment id e.g. /api/articles/banana/comments